pinot-controller/src/main/resources/static/query/index.html

```diff
@@ -79,11 +79,11 @@ <h1><a href="#/">Pinot Data Explorer</a></h1>
           <div class="small-2 columns" title="Enables tracing">
             <input type=checkbox id="trace-enabled">&nbsp;&nbsp;Tracing</input>
           </div>
-          <div class="small-3 columns" title="Sets groupByMode=sql in queryOptions (default pql)">
```

Comment:
We need only 1 checkbox now for "PQL", which will go to the "/pql" endpoint. By default we go to "sql" endpoint

Reply:
changed the ui to only show one check box of pql, which means to use pql syntax if checked.